data/json/professions.json

```diff
@@ -4026,6 +4026,18 @@
       "female": [ "bra", "boy_shorts" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "kit_survivor",
+    "name": "Kitted Survivor",
+    "description": "At the local mall, you saw a sign advertising a discount on survival kits. You bought one, more for show than for actual use. Now, it's all you have."
```

Comment:
```suggestion
    "description": "At the local mall, you saw a sign advertising a discount on survival kits. You bought one, more for show than for actual use. Now, it's all you have.",
```

Reply:
Oh - how do I lint my JSON?

Reply:
Lint the whole file contents with a http://dev.narc.ro/cataclysm/format.html